.travis.yml

```diff
@@ -17,12 +17,20 @@ addons:
     organization: adobeinc
 services:
 - xvfb
-script:
-- npm install
-- npm run build:prod
-- npm run functional:ci
-- npm run test:saucelabs
-- npm run checkbundlesize
+
+jobs:
+  - name: test integration env
+    script:
+    - npm install
+    - npm run build:prod
+    - npm run functional:ci
+    - npm run test:saucelabs
+    - npm run checkbundlesize
+  - name: test production env
+    script:
+    - npm install
+    - npm run functional:local:prod
```

Comment:
Should we be using SauceLabs like we are for `npm run functional:ci` (I might not understand what's going on here)? Should this npm script be named something  other than `*:local:*` (or essentially alias the one that says `*:local:*`)?

Reply:
Yes sir for sure, SauceLabs and rename the script. I needed a quick local test to confirm the CRON is working. Once I confirm it, I will update the info. Thanks!